src/core/server/app/handlers/api/account/download.ts

```diff
@@ -216,6 +218,46 @@ export const downloadHandler = ({
   };
 };
 
+export const adminDownloadHandler = ({
+  mongo,
+  signingConfig,
+}: AdminDownloadOptions): RequestHandler => {
+  return async (req: Request, res, next) => {
```

Comment:
the only difference between this method and the user-facing one is that it doesn't have the rate-limiting, I'm not mad about the duplicated code, but I could understand if someone was!